elasticdl/python/worker/worker.py

```diff
@@ -350,6 +363,47 @@ def report_gradient_to_ps(self, grads):
             req.model_version = self._model_version
             res = self._ps_stubs[ps_id].push_gradient(req)
 
+        if self._embedding_layers:
```

Comment:
Here we should combine non-embedding param gradients and embedding gradients in one `push_gradient` call.
This is because in ps `push_gradient` service, each `push_gradient` would increase the parameter version by 1. 
Two calls would update the version by 2.

Reply:
Done